homeassistant/components/unifi/controller.py

```diff
@@ -67,7 +66,7 @@
 from .errors import AuthenticationRequired, CannotConnect
 
 RETRY_TIMER = 15
-CHECK_DISCONNECTED_INTERVAL = timedelta(seconds=1)
+CHECK_HEARTBEAT_INTERVAL = timedelta(seconds=1)
```

Comment:
Is it important to run the check every second?

Reply:
That was my thought as well, but I figured it would likely be a breaking change to alter the behavior since the time is configurable. 

Since there are places that are at least an order of magnitude less performant (writing state 18k times per minute), I'm going to focus on those since this no longer shows up in the top of profile.